framework/tlvf/tlvf.py

```diff
@@ -985,7 +985,7 @@ def addClassVarLenMethods(self, obj_meta, param_type, param_name, param_meta, pa
             lines_cpp.append( "%s}" % self.getIndentation(1) )
             lines_cpp.append( "%sm_%s__ = %s;" %(self.getIndentation(1), self.MEMBER_LOCK_ORDER_COUNTER, param_meta.list_index) )
             lines_cpp.append( "%sm_%s__ = true;" % (self.getIndentation(1), self.MEMBER_LOCK_ALLOCATION) )
-            lines_cpp.extend(self.addAllocationMarkers(obj_meta, "create", param_meta, param_length, True)) # Variable length lists support
+            lines_cpp.extend(self.addAllocationMarkersCreate(obj_meta, "create", param_meta, param_length, True)) # Variable length lists support
```

Comment:
If you split to 3 separate functions - no need for the second argument. Why split in the first place?

Reply:
because for each one of: "create", "alloc", "add" has a different way to do it, and it is more convenient to find it int the tlvf.